bin/create_tracks

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Do you know a nice way of dealing with platforms that don't have provide the python2 command yet?